src/js/yeldaWebchatInjector.js

```diff
@@ -158,25 +158,43 @@ class YeldaChat {
         return
       }
 
-      const customImage =  settings.data.image && settings.data.image.url
+      const isVoiceFirstUI = settings.data.hasOwnProperty('isVoiceFirstUI') || false
+      const customImage = settings.data.image && settings.data.image.url
       const hasCustomStyle = settings.data.hasOwnProperty('isDefaultStyle') && !settings.data.isDefaultStyle
 
-      if (!hasCustomStyle || !customImage) {
-        this.webChatContainer.appendChild(this.assistantImage)
+      /**
+       * in isVoiceFirstUI mode
+       * - the assistant should be opened directly and will never be closed
+       * => we call directly openChat and do not add the assistant image
+       * - we don't want the box-shadow css style
+       *    => we add voiceFirstUI to iframeContainer
+       * - /chat vue render the voice first UI
+       *   => nothing more to do here
+       */
+      if (isVoiceFirstUI) {
+        // voiceFirstUI added to the iframeContainer to remove box-shadow css style
+        // other styles can be added for voice first UI based on this class in the future if needed
+        this.iframeContainer.classList.add('voiceFirstUI')
```

Comment:
Can you add the needed test for this please ?

- if isVoiceFirstUI is true, voiceFirstUI is added to the iframeContainer
- if isVoiceFirstUI is false, voiceFirstUI is not added to the iframeContainer

- if isVoiceFirstUI is true, assistantImage is not added to the webChatContainer
- if isVoiceFirstUI is false, assistantImage is added to the webChatContainer